src/ui/templates.js

```diff
@@ -0,0 +1,32 @@
+import moment from 'moment'; // TODO see below
+import type { MessageDetails, MessageEntry } from './types';
+import { messageToDetails } from './transform';
+import { byTimeDescending } from '../utility/misc';
+
+// TODO how to resolve format here? importing moment doesn't
+export const detailsTemplate = (details: MessageDetails) => {
```

Comment:
I might be lazier than most, but as soon as I’m typing `details.` twice I switch to destructuring.